### PR TITLE
File import

### PR DESCRIPTION
**Pull request by kambed**

### CODE REVIEW

File: src/main/java/FileOperator.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            int i = 0;
+            int j = 0;
+            for (String s : lines) {
+                for (String arg : s.split("\\s+")) {
+                    board[i][j] = Integer.parseInt(arg);
+                    j++;
+                }
+                j = 0;
+                i++;
+            }
```
Comment (RadekBucki) [inline src/main/java/FileOperator.java]: if you need `i` and `j`, it should be better to use clasic for (`for(int i = 0; i<lines.length; i++)`).

File: src/main/resources/equation1.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+3 3 1 12
+2 5 7 33
+1 2 1 8
```
Comment (RadekBucki) [inline src/main/resources/equation1.txt:3]: Add input file to `.gitignore`

**Review by RadekBucki — CHANGES_REQUESTED** [review]

File: src/main/resources/equation1.txt (hunk 2 above)
Comment (kambed) [reply src/main/resources/equation1.txt:3]: boards are the part of the exercise so it can be placed on github as resource

Comment (RadekBucki) [reply src/main/resources/equation1.txt:3]: Board isn't resource. Resources are required to use program. For now you Putin one of many example, which can be solved by program in to JVM and require specific path for matrix. It is uregulowania online in IDE, not in general.

**Pull request closed without merging**